pkg/crd/schema.go

```diff
@@ -254,7 +254,7 @@ func localNamedToSchema(ctx *schemaContext, ident *ast.Ident) *apiext.JSONSchema
 	}
 	// NB(directxman12): if there are dot imports, this might be an external reference,
```

Comment:
Thx! 

Please add the following so we have test coverage for aliases in general

https://github.com/kubernetes-sigs/controller-tools/blob/7ef21aae5452047af82f020774639c94aa659eca/pkg/crd/testdata/cronjob_types.go#L329

```go
	// This tests that string alias is handled correctly.
	StringAlias StringAlias `json:"stringAlias,omitempty"`
```

https://github.com/kubernetes-sigs/controller-tools/blob/7ef21aae5452047af82f020774639c94aa659eca/pkg/crd/testdata/testdata.kubebuilder.io_cronjobs.yaml#L8975

```go
              stringAlias:
                type: string
```

Reply:
Thanks for the tips, just added the test case and checked it worked!

Reply:
ping @sbueringer :)